domain-models-runtime/src/main/java/org/folio/rest/tools/utils/NetworkUtils.java

```diff
@@ -16,6 +16,10 @@
  */
 public class NetworkUtils {
 
+  /**
+ *  ThreadLocalRandom has a Random instance per thread and safeguards against contention.
```

Comment:
Is this a comment about the method or about the tag `@SuppressWarnings`? If the latter it is more easy to understand for me if moved it into a `//` comment into the tag line.